README.md

```diff
@@ -29,111 +30,46 @@ Check out the Getting Started guide in our [docs](https://astronomer.github.io/a
 
 ## Sample DAGs
 
-```python
-from datetime import datetime, timedelta
-from pathlib import Path
-
-from airflow import DAG
-
-from ray_provider.operators.ray import SetupRayCluster, SubmitRayJob, DeleteRayCluster
-
-default_args = {
-    "owner": "airflow",
-    "start_date": datetime(2024, 3, 26),
-    "retries": 1,
-    "retry_delay": timedelta(minutes=0),
-}
+### Example 1: Using @ray.task for job life cycle
+The below example showcases how to use the ``@ray.task`` decorator to manage the full lifecycle of a Ray cluster: setup, job execution, and teardown.
 
-
-RAY_SPEC = Path(__file__).parent / "scripts/ray.yaml"
-FOLDER_PATH = Path(__file__).parent / "ray_scripts"
-
-dag = DAG(
-    "Setup_Teardown",
-    default_args=default_args,
-    description="Setup Ray cluster and submit a job",
-    schedule=None,
-)
-
-setup_cluster = SetupRayCluster(
-    task_id="SetupRayCluster",
-    conn_id="ray_conn",
-    ray_cluster_yaml=str(RAY_SPEC),
-    use_gpu=False,
-    update_if_exists=False,
-    dag=dag,
-)
-
-submit_ray_job = SubmitRayJob(
-    task_id="SubmitRayJob",
-    conn_id="ray_conn",
-    entrypoint="python script.py",
-    runtime_env={"working_dir": str(FOLDER_PATH)},
-    num_cpus=1,
-    num_gpus=0,
-    memory=0,
-    resources={},
-    fetch_logs=True,
-    wait_for_completion=True,
-    job_timeout_seconds=600,
-    xcom_task_key="SetupRayCluster.dashboard",
-    poll_interval=5,
-    dag=dag,
-)
-
-delete_cluster = DeleteRayCluster(
-    task_id="DeleteRayCluster",
-    conn_id="ray_conn",
-    ray_cluster_yaml=str(RAY_SPEC),
-    use_gpu=False,
-    dag=dag,
-)
-
-# Create ray cluster and submit ray job
-setup_cluster.as_setup() >> submit_ray_job >> delete_cluster.as_teardown()
-setup_cluster >> delete_cluster
-```
+This approach is ideal for jobs that require a dedicated, short-lived cluster, optimizing resource usage by cleaning up after task completion
 
 ```python
```

Comment:
@venkatajagannath would it be possible to have these examples embedding the code from `example_dags` folder? This would help us not have them outdated and automatically tested by the CI.

This is an example how we did this in another Astronomer project:
https://github.com/astronomer/astro-sdk/blob/main/README.md

Reply:
Yes, I think we should render it from an example code I also requested this https://github.com/astronomer/astro-provider-ray/pull/30#discussion_r1705766724. You can either add the commit url for code block or you can can convert readme to rst

Reply:
@tatiana @pankajastro 

I tried doing it.. but it does not render correctly.. can you please take a look?

Reply:
updated